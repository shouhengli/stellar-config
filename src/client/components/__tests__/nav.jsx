const React = require('react');
const Nav = require('../nav.jsx');
const renderer = require('react-test-renderer');

describe('component nav', () => {
  test('wraps its children', () => {
    const component = renderer.create(<Nav>content</Nav>);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
