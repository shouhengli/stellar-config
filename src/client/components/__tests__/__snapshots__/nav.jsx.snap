// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component nav wraps its children 1`] = `
<nav
  className="navbar is-fixed"
>
  <div
    className="navbar-brand"
  >
    <div
      className="navbar-item"
    >
      SEAWEED
    </div>
  </div>
  <div
    className="navbar-menu is-active"
  >
    content
  </div>
</nav>
`;
